CHANGELOG.md

```diff
@@ -67,6 +67,7 @@ memory error
 - Fixed nif_atomvm_posix_read GC bug
 - Fixed `erlang:is_number/1` function, now returns true also for floats
 - Fixed unlink protocol and add support for `link/1` on ports
+- Correctly set Pico-W unique hostname, use `dhcp_hostname` if present in config.
```

Comment:
I think we should at least add also an entry to "Added" in order to mention that now we allow to configure the hostname (using a certain option).
While under "Fixed" I would just focus on what has been fixed (I don't find this entry really clear).